# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/bsd-games/delay_output-sym.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN bsd-games-2.16/hack/extern.h bsd-games-2.16-new/hack/extern.h
--- bsd-games-2.16/hack/extern.h	2004-01-27 21:51:11.000000000 +0100
+++ bsd-games-2.16-new/hack/extern.h	2005-05-17 23:16:15.000000000 +0200
@@ -511,7 +511,7 @@
 void standoutend(void);
 void backsp(void);
 void bell(void);
-void delay_output(void);
+void delay_output_my(void);
 void cl_eos(void);
 
 /* hack.timeout.c */
diff -ruN bsd-games-2.16/hack/hack.pri.c bsd-games-2.16-new/hack/hack.pri.c
--- bsd-games-2.16/hack/hack.pri.c	2004-01-26 03:08:30.000000000 +0100
+++ bsd-games-2.16-new/hack/hack.pri.c	2005-05-17 23:16:15.000000000 +0200
@@ -171,7 +171,7 @@
 		return;
 	}
 	if (prevx >= 0 && cansee(prevx, prevy)) {
-		delay_output();
+		delay_output_my();
 		prl(prevx, prevy);	/* in case there was a monster */
 		at(prevx, prevy, levl[prevx][prevy].scrsym);
 	}
@@ -218,7 +218,7 @@
 	/* normal call */
 	if (cansee(x, y)) {
 		if (cnt)
-			delay_output();
+			delay_output_my();
 		at(x, y, let);
 		tc[cnt].x = x;
 		tc[cnt].y = y;
diff -ruN bsd-games-2.16/hack/hack.termcap.c bsd-games-2.16-new/hack/hack.termcap.c
--- bsd-games-2.16/hack/hack.termcap.c	2003-12-17 03:46:37.000000000 +0100
+++ bsd-games-2.16-new/hack/hack.termcap.c	2005-05-17 23:16:15.000000000 +0200
@@ -326,7 +326,7 @@
 }
 
 void
-delay_output()
+delay_output_my()
 {
 
 	/* delay 50 ms - could also use a 'nap'-system call */
